models/challenges.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * This file contains wrapper functions to interact with the DB.
+ * This will contain the DB schema if we start consuming an ORM for managing the DB operations
+ */
+
+const Firestore = require('@google-cloud/firestore')
+const firestore = require('../utils/firestore')
+
+const challengesModel = firestore.collection('challenges')
+const userModel = firestore.collection('users')
+
+const cannotSubscribe = 'User cannot be subscribed to challenge'
+const userDoesNotExistError = 'User does not exist. Please register to participate'
+const errorMessage = 'Error getting challenges'
```

Comment:
NIT: We should capitalize the constant variables

Reply:
Will do that.